packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -103,10 +103,25 @@ export const ContextualMenuItem = compose<ContextualMenuItemType>({
       root: {
         ...pressable.props,
         ref: cmRef,
-        onKeyUp: onKeyUp,
-        onMouseEnter: onMouseEnter,
-        onMouseLeave: onMouseLeave,
+        onKeyUp,
+        onMouseEnter,
+        onMouseLeave,
+        accessible: true,
         accessibilityLabel: accessibilityLabel,
+        accessibilityRole: 'menuitem',
+        accessibilityState: { disabled: state.disabled, selected: state.selected },
```

Comment:
Actually, @lenahong I just tested on win32, and setting accessibilityState here means that if the menu item is disabled, it will not receive keyboard focus. Is that acceptable behavior?

Reply:
I checked and that is the expected behavior on win32 as well! Thanks for checking with me.